configs/model/fly_tethered.yaml

```
@@ -0,0 +1,180 @@
+# MJCF_PATH: 'models/fruitfly/fruitfly_freeforce.xml'
+MJCF_PATH: 'models/fruitfly/fruitfly_force.xml'
+
+# Frames per clip for transform.
+N_FRAMES_PER_CLIP: 300
+
+# Tolerance for the optimizations of the full model, limb, and root. 
```

yamllint

[error] trailing spaces

(trailing-spaces)